<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>615</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: arial;
	font-size: 12px;
}

</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{background: rgba(0,0,0,0.8);
	border-radius: 15px;}
QLineEdit {
	background: transparent;
	border: none;
	border-bottom: 1px solid #717072;
}
#stackedWidget
{
background: url(:/nowyPrzedrostek/mountains.jpg);}</string>
   </property>
   <widget class="QWidget" name="page1">
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color :white;
	background: red;
	border-radius: 25px;
    border-radius: 3px;
}

QPushButton:hover {
	background: wihte;
	color: red;
}</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>130</y>
       <width>500</width>
       <height>400</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Expenses</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Income</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#717072;</string>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#717072;</string>
      </property>
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	background: transparent;
	border: none;
	color: #717072;
	border-bottom: 1px solid #717072;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>481</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(104, 104, 104);
font: 700 9pt &quot;Arial&quot;;
color:black;
border-radius: 15px;

border-bottom: 1px solid #717072;</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>I/E</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Amount</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DELETE ROW</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CLEAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PIE CHART</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>360</y>
        <width>90</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LOGOUT</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>15</y>
       <width>500</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hi, {name}. Welcome to your profile!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your balance is: {balance}</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
